VimeoNetworking/VimeoNetworking/NSError+Extensions.swift

```diff
@@ -233,56 +252,37 @@ public extension NSError
         return nil
     }
     
-        /// Returns an array of error codes from the api error JSON dictionary if any exist, otherwise returns an empty array
-    public var vimeoInvalidParametersErrorCodes: [Int] {
-        
-        var errorCodes: [Int] = []
-        
-        guard let json = self.errorResponseBodyJSON else {
-            return errorCodes
-        }
-        
-        guard let invalidParameters = json[self.dynamicType.VimeoInvalidParametersKey] as? NSArray else {
-            return errorCodes
-        }
-        
-        for errorJSON in invalidParameters {
-            if let errorCode = errorJSON[self.dynamicType.VimeoErrorCodeKey] as? Int {
-                errorCodes.append(errorCode)
-            }
-        }
-        
-        return errorCodes
-    }
-    
         /// Returns the first error code from the api error JSON dictionary if it exists, otherwise returns NSNotFound
     public var vimeoInvalidParametersFirstErrorCode: Int
     {
         return self.vimeoInvalidParametersErrorCodes.first ?? NSNotFound
     }
     
-    public var vimeoInvalidParametersFirstVimeoUserMessage: String? {
-        
-        guard let json = self.errorResponseBodyJSON,
-            invalidParameters = json[self.dynamicType.VimeoInvalidParametersKey] as? NSArray else {
+        /// Returns the user message from the api error JSON dictionary if it exists
+    public var vimeoInvalidParametersFirstVimeoUserMessage: String?
+    {
+        guard let json = self.errorResponseBodyJSON, invalidParameters = json[self.dynamicType.VimeoInvalidParametersKey] as? [AnyObject] else
```

Comment:
I suppose it's out of scope to covert all these `self.dynamicType`s to a Constants struct, eh? Can we file a ticket in the tech debt backlog for it?